app/wait-for-local-db.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# wait-for-local-db
+
+set -e
+
+# Color formatting
+RED='\033[0;31m'
+NO_COLOR='\033[0m'
+
+MAX_WAIT_TIME=30 # seconds
+wait_time=0
+
+# Use pg_isready to wait for the DB to be ready to accept connections
+# We check every 3 seconds and consider it failed if it gets to 30+
+# https://www.postgresql.org/docs/current/app-pg-isready.html
+while ! pg_isready -h localhost -d main-db -q;
```

Comment:
optional suggestion: what do you think about using the `until` command the way VA uses it (https://github.com/department-of-veterans-affairs/abd-vro/blob/e88312e8d35c71013dcd8748b123f2c77f9de59b/helm/vro-app/templates/deployment.yaml#L38-L41)

i hadn't known about that command before but it reads pretty nicely. i don't feel strongly either way though

Reply:
I like that, will switch.